web/src/app/apollo.js

```diff
@@ -100,6 +100,14 @@ const simpleCacheTypes = [
 // tell Apollo to use cached data for `type(id: foo) {... }` queries
 const queryCache = {}
 
+const cache = new InMemoryCache({
+  cacheRedirects: {
+    Query: {
+      ...queryCache,
+    },
+  },
+})
+
 simpleCacheTypes.forEach(name => {
   queryCache[camelCase(name)] = (_, args) =>
```

Comment:
We'll have to be careful here, iirc it was a little awkward and order does matter here because we are defining `queryCache = {}`, then iterating over `simpleCacheTypes` to set the keys in `queryCache`, then creating the `cache` from that.

We may need to ignore the rule in this file if moving the `cache` def above causes issues